spyder_line_profiler/spyder/widgets.py

```diff
@@ -613,23 +770,74 @@ def populate_tree(self):
 
                 # Monospace font for code
                 line_item.setFont(COL_LINE, monospace_font)
+                
+    def onItemClick(self, item):
```

Comment:
Following the suggestion above:
```suggestion
    def on_item_clicked(self, item):
```